File: Jaewon/Level.1/39) 예산/39.js
function solution(d, budget) {
    d.sort((a,b) => a-b);
    let i = 0, j = 0;
    while(i < d.length){
        j += d[i];
        if(j > budget)
            break;
        i++;
    }
    return i;
}

File: Jaewon/Level.1/39) 예산/39.cpp
#include <iostream>
#include <stdio.h>
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(vector<int> d, int budget) {
    sort(d.begin(), d.end());
    
    int i = 0;
    for(i; i<d.size(); i++)
    {
        if(budget - d[i] >= 0)
            budget -= d[i];
        else break;
    }
    
    return i;
}